### ISSUE TITLE
LayoutLMv2 RGB or BGR?

### ISSUE THREAD

**Issue by aripo99** [NielsRogge/Transformers-Tutorials#96]
Hi! 

I was wondering if we are supposed to pass in the images to the LayoutLMv2 processor in 'RGB' or 'BGR' format? From the documentation, I thought we were supposed to use BGR but I have noticed that in your CORD notebook you convert the image to RGB.

Thank you so much :)

**Comment by NielsRogge** [comment]
Hi :)

LayoutLMv'2 expects BGR (as all models in Detectron2 are trained with the BGR format).

From the [docs](https://huggingface.co/docs/transformers/model_doc/layoutlmv2#overview):

> Important to note is that the visual backbone expects BGR channels instead of RGB, as all models in Detectron2 are pre-trained using the BGR format.

**Comment by AJHoeh** [comment]
I just stumbled over this aswell after preprocessing the whole rvl-cdip for ~50 hours :sweat_smile:  
You might want to consider updating the LayoutLMv2 Notebooks, as far as I can tell they all use RGB format

**Comment by YangjiaqiDig** [comment]
> I just stumbled over this aswell after preprocessing the whole rvl-cdip for ~50 hours 😅 @NielsRogge you might want to consider updating the LayoutLMv2 Notebooks, as far as I can tell they all use RGB format

@NielsRogge 
Agree, should you doubel check the v2 tutorial notebook? in prepare_examples, all images are read in 'RGB' from PIL.Image
